pkg/webui/console/components/dev-eui-component/index.js

```diff
@@ -0,0 +1,183 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import classnames from 'classnames'
+
+import Input from '@ttn-lw/components/input'
+import Form from '@ttn-lw/components/form'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import tooltipIds from '@ttn-lw/lib/constants/tooltip-ids'
+import env from '@ttn-lw/lib/env'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+import PropTypes from '@ttn-lw/lib/prop-types'
+import attachPromise from '@ttn-lw/lib/store/actions/attach-promise'
+
+import { getApplicationDevEUICount, issueDevEUI } from '@console/store/actions/applications'
+
+import {
+  selectApplicationDevEUICount,
+  selectSelectedApplicationId,
+} from '@console/store/selectors/applications'
+
+import style from './dev-eui.styl'
+
+const DevEUIComponent = props => {
+  const { values, setFieldValue, initialValues, devEUISchema } = props
```

Comment:
```suggestion
  const { devEUISchema } = props
  const { values, setFieldValue, initialValues } = useFormContext()
```
If you are setting this up as a `<Form.Field />` then you can assume that this component will be rendered within a `<Form />` in which you will have access to the form context.
Also, [you were not passing any of these props anyway](https://github.com/TheThingsNetwork/lorawan-stack/pull/5594/files#diff-b5fea30b9eed7fac720e3f08220be51470dc6a0cc19546b9699db48f1bbdf1a7R33).